lib/pengine/pe_digest.c

```diff
@@ -205,12 +205,11 @@ is_fence_param(xmlAttrPtr attr, void *user_data)
  * \param[in]  params      Resource parameters evaluated for node
  * \param[in]  xml_op      XML of operation in CIB status (if available)
  * \param[in]  op_version  CRM feature set to use for digest calculation
- * \param[in]  overrides   Key/value hash table to override resource parameters
  */
 static void
 calculate_secure_digest(op_digest_cache_t *data, pe_resource_t *rsc,
                         GHashTable *params, xmlNode *xml_op,
-                        const char *op_version, GHashTable *overrides)
+                        const char *op_version)
```

Comment:
Since we are now using data->params_all here, it has already taken overrides into account with the previous invocation of calculate_main_digest(). Or am I missing anything?

Reply:
Oh right, I overlooked that.